genesis-builder/framework/evm/sources/Evm.move

```diff
@@ -0,0 +1,75 @@
+module Evm::evm {
+    use aptos_framework::fungible_asset::{Self, FungibleAsset};
+    use aptos_framework::primary_fungible_store::ensure_primary_store_exists;
+    use EthToken::eth_token::get_metadata;
+    use std::error;
+    use std::signer;
+
+    /// For now deploying EVM contracts is restricted to an admin account.
+    /// This restriction may be lifted in the future.
+    const ENOT_OWNER: u64 = 1;
+
+    const OWNER: address = @0x1;
```

Comment:
We can actually define this as a variable in the `Move.toml` and refer to it as `const OWNER: address = @owner;`

Reply:
Fixed in https://github.com/MovedNetwork/op-move/pull/143/commits/d0d84791b2756844ac7beab0314db5e56d56e451